src/utils/semantics.utils.ts

```diff
@@ -0,0 +1,31 @@
+import { blanksDefaultTranslations, defaultTranslations, dragTextDefaultTranslations } from '../constants/defaultTranslations';
+import { AnswerModeType, type Params } from '../types/types';
+
+export const getDefaultParams = (): Required<Params> => {
+  return {
+    description: undefined,
```

Comment:
`description`, `words` and `overallFeedback` are optional, so they don't need to be sanitized. Setting them explicitly doesn't hurt though - in particular since `InferParamsFromSemantics` from the `h5p-types` package doesn't seem to make this distinction and doesn't set the properties as optional. Is this something that Sindre might want to change?